vcx/wrappers/node/src/vcx.ts

```diff
@@ -35,6 +34,6 @@ export class VCXRuntime {
     const libDir = libPath[platform.toLowerCase()] || libPath['linux']
     const library = `libvcx${postfix}`
     const customPath = process.env.LIBVCX_PATH ? process.env.LIBVCX_PATH + library : undefined
-    return customPath || this._config.basepath || path.resolve(__dirname, `${libDir}${library}`)
+    return customPath || this._config.basepath || `${libDir}${library}`
```

Comment:
The line `path.resolve(__dirname, `${libDir}${library}`)` was not making sense as the second parameter for `path.resolve` was always absolute, and `path.resolve` will in that case ignore the first argument.
```
> path.resolve("./relative", "/bar/foo")
'/bar/foo'
> path.resolve("/absolute", "/bar/foo")
'/bar/foo'
```